frontend/src/__tests__/basic.test.js

```diff
@@ -0,0 +1,246 @@
+        const { payload: withNewPosition } = await apiGET(
+            `/sessions/${createdSessionId}/positions`
+        );
+
+        expect(withNewPosition.length).toBe(initialPositions.length + 1);
+        // make sure the id of our new position came back
+        expect(withNewPosition.map(x => x.id)).toContain(createdPosition.id);
+
+        // save the id for later use
+        createdPositionId = createdPosition.id;
+    });
+
+    it("create an applicant", async () => {
+        const { payload: initialApplicants } = await apiGET(
+            "/admin/applicants"
+        );
+
+        // check that there is no Applicant before the create is run
+        expect(initialApplicants).toMatchObject([]);
+
+        const resp1 = await apiPOST(`/admin/applicants`, newApplicantData);
+
+        // check if we get a success response
+        expect(resp1).toMatchObject({ status: "success" });
+
+        const { payload: createdApplicant } = resp1;
+
+        // make sure the propTypes are right
+        checkPropTypes(applicantPropTypes, createdApplicant);
+        // make sure the data we get back is the same we put in
+        expect(createdApplicant).toMatchObject(newApplicantData);
+        // make sure we have an id
+        expect(createdApplicant.id).not.toBeNull();
+
+        // fetch all Applicants and make sure the data is in there
+        const { payload: withNewApplicant } = await apiGET("/admin/applicants");
+
+        expect(withNewApplicant.length).toBe(initialApplicants.length + 1);
+        // make sure the id of our new Applicant came back
+        expect(withNewApplicant.map(x => x.id)).toContain(createdApplicant.id);
+
+        // save the id for later use
+        createdApplicantId = createdApplicant.id;
+    });
+
+    it("create an assignment", async () => {
+        const newAssignmentData = {
+            position_id: createdPositionId,
+            applicant_id: createdApplicantId,
+            start_date: new Date("2019/09/09").toISOString(),
+            end_date: new Date("2019/12/31").toISOString(),
+            hours: 90
+        };
+
+        const resp1 = await apiPOST(`/admin/assignments`, newAssignmentData);
+
+        // check if we get a success response
+        expect(resp1).toMatchObject({ status: "success" });
+
+        const { payload: createdAssignment } = resp1;
+
+        // make sure the propTypes are right
+        checkPropTypes(assignmentPropTypes, createdAssignment);
+        // make sure the data we get back is the same we put in
+        expect(createdAssignment).toMatchObject(newAssignmentData);
+        // make sure we have an id
+        expect(createdAssignment.id).not.toBeNull();
+
+        // fetch the new assignment we just created
+        const newAssignment = await apiGET(
+            `/admin/assignments/${createdAssignment.id}`
+        );
+
+        expect(newAssignment.status).toBe("success");
+        expect(newAssignment.payload).toMatchObject(newAssignmentData);
+    });
```

Comment:
Group all of these tests into a `"seed database"` test. Seed everything and assume it works. Then, at the end of the test, retrieve everything and verify that it is our seed data.